lib/MainTemplate.js

```diff
@@ -253,11 +248,40 @@ module.exports = class MainTemplate extends Tapable {
 			buf.push(`${this.requireFn}.r = function(exports) {`);
 			buf.push(
 				Template.indent([
+					"if(typeof Symbol !== 'undefined' && Symbol.toStringTag) {",
```

Comment:
Doesn't symbol break backwards IE compat?

Reply:
If so we'd need make a breaking change.

Reply:
FWIW polyfilling `object.assign` for IE was all I needed to get it to function.

Reply:
This PR is not only about Object.assign. It's about creating more spec compilant namespace objects for ESM and non-ESM interop.

Reply:
Sorry I missed the typeof when I initially saw this.